native/machine-coding/Weather App/index.js

```diff
@@ -0,0 +1,48 @@
+// API TO BE USED FOR THE WEATHER DETAILS
+const apiKey = "46d47581a51a79782741111953e700af";        
+const apiUrl = "https://api.openweathermap.org/data/2.5/weather?units=metric&q=";
+
+const searchBox = document.querySelector(".search input");
+const searchBtn = document.querySelector(".search button");
+const weatherIcon = document.querySelector(".weather-icon");
+
+async function checkWeather(city){
+    const response = await fetch(apiUrl + city + `&appid=${apiKey}`);   //to fetch the details when city name is entered.
+
+    if(response.status == 404){                                         // when city name is entered wrong
+        document.querySelector(".error").style.display = "block";
+        document.querySelector(".weather").style.display = "none";
+    }
+
+    else{
+    var data = await response.json();
+   
+    document.querySelector(".city").innerHTML = data.name;
+    document.querySelector(".temp").innerHTML = Math.round(data.main.temp) + "°C";
+    document.querySelector(".humidity").innerHTML = data.main.humidity + "%";
+    document.querySelector(".wind").innerHTML = data.wind.speed + "km/h";
+
+    if(data.weather[0].main == "Clouds"){
+        weatherIcon.src = "images/clouds.png"
+    }
+    else if(data.weather[0].main == "Clear"){
+        weatherIcon.src = "images/clear.png"
+    }
+    else if(data.weather[0].main == "Rain"){
+        weatherIcon.src = "images/rain.png"
+    }
+    else if(data.weather[0].main == "Drizzle"){
+        weatherIcon.src = "images/drizzle.png"
+    }
+    else if(data.weather[0].main == "Mist"){
+        weatherIcon.src = "images/mist.png"
+    }

```

Comment:
can we not directly use the value of man here

weatherIcon.src = `images/${data.weather[0].main.toLowerCase()}.png`

Reply:
So i tried doing this but it was as it was not properly functional thus i didnot made this change